csrankings-g.csv

```diff
@@ -646,6 +646,7 @@ Greg N. Frederickson,Purdue University,https://www.cs.purdue.edu/people/gnf,NOSC
 Greg Shakhnarovich,TTI Chicago,http://ttic.uchicago.edu/~gregory,YLOz1kgAAAAJ
 Greg Speegle,Baylor University,https://www.ecs.baylor.edu/index.php?id=961989,B5KUbnAAAAAJ
 Greg Turk,Georgia Institute of Technology,http://www.cc.gatech.edu/~turk,Q_4d9N0AAAAJ
+Greg Ver Steeg,Univ. of California - Riverside,https://www.isi.edu/directory/gregv/,goLucoIAAAAJ

```

Comment:
"Research Associate Professor" = not tenure-track, I assume, and so, not eligible for inclusion.

Reply:
I am 99% sure he is being hired as a (not research) associate professor.  Let me confirm and report back

Reply:
I have confirmed Greg will be joining as an Associate Professor.  I will ask him to update his web site